docker-local-mojaloop/README.md

```diff
@@ -0,0 +1,389 @@
+# Local Mojaloop + Custom DFSP Test setup
+
+The aim of this Document is to have a local mojaloop running with two customs DFSPs (backend plus sdk-adapter) and a PISP represented by a mojaloop simulator (which implements mojaloop api).
+
+
+## Create host
+
+* Add to /etc/hosts
+```
+127.0.0.1       central-ledger.local central-settlement.local ml-api-adapter.local account-lookup-service.local account-lookup-service-admin.local quoting-service.local moja-simulator.local central-ledger central-settlement ml-api-adapter account-lookup-service account-lookup-service-admin quoting-service simulator host.docker.internal
+```
+
+## Start the compose
+
+
+```
+Running the services separately in different terminals is recommended
+
+Run
+
+docker-compose up account-lookup-service
+
+Then close once the service is ready.
+Proceed to stop account-lookup-service and proceed with the following commands.
+ALS needs to be run initially so it setups properly.
+
+docker-compose up central-ledger
+docker-compose up quoting-service
+docker-compose up ml-api-adapter
+docker-compose up central-settlement
+docker-compose up account-lookup-service
+docker-compose up dfspa-scheme-adapter dfspa-backend
+docker-compose up dfspb-scheme-adapter dfspb-backend
+```
+
+## Create some initial data
+
+### Prerequisites
+
+* Install newman
+
+```
+npm install -g newman
+```
+
+* cd to the postman folder
+
+* Use this to run all the scripts
+
+```
+sh scripts/setupDockerCompose-FullSetup.sh
+```
+
```

Comment:
here should be a not that if we run `sh scripts/setupDockerCompose-FullSetup.sh` there is no need to run other setups scripts

Reply:
changed copy to "Use this convenience script to run all the setup scripts and forgo having to run the setup scripts individually." 

hope that clarifies it a bit. thanks.